docs/presentations/articles/OS_LLMs_Robert.md

```diff
@@ -0,0 +1,278 @@
+| 🏆 In September 2023, the TII unveiled the big brother of the Falcon series with **180B** parameters, which, upon its introduction, achieved the highest rankings in most benchmarks. The key to its success lay in its high-quality dataset, which was also released as open source. | ![Alt text](https://gulfbusiness.com/wp-content/uploads/2023/06/UAEs-Falcon-40B-AI-Model-is-Now-Royalty-free-image-credit-Technology-Innovation-Institute.png) |
+
+#### 2.1 ) The dataset
+
+<div style="text-align:center">
+  <img src="./assets/os-llms-falcon-dataset.png" alt="Alt text" width="500"/>
+</div>
+
+The new dataset **RefinedWeb** represents an innovative, comprehensive web dataset based on CommonCrawl. Falcon enhances this data through deduplication and rigorous content filtering.
+In addition to RefinedWeb, books, conversations, code, and technical-scientific papers were also utilized.
+
+The model was primarily trained in **English**, but 7% of the dataset was expanded to **European languages**.
+
+<div style="text-align:center">
+  <img src="./assets/os-llms-falcon-refinedweb-eu.png" alt="Alt text" width="500"/>
+</div>
+
+!!! info Required Resources
+⚠️ The significance of the parameter count becomes apparent as the training of such models is costly and resource-intensive.
+
+    **7B Model**
+    - Training: 384x A100 40GB GPUs for 2 weeks
+    - Usage: minimum 16 GB GPU-RAM
+
+    **40B Model**
+    - Training: 384x A100 40GB GPUs for 2 months
+    - Usage: minimum 90 GB GPU-RAM
+
+    **180B Model**
+    - Training: 4,069x A100 40GB GPUs (AWS)
+
+<br><br>
+
+---
+
+## 3) LLaMa
+
+Another success among open-source LLM models is the Llama model by Meta.
+
+<div style="text-align:center">
+  <img src="https://mixed.de/wp-content/uploads/2021/11/Meta_Logo.jpg" alt="Alt text" width="200"/>
+</div>
+
+#### 3.1) LLaMA 1
+
+The first Llama model, released on 24.02.2023, achieves performance comparable to OpenAI's GPT-3 model.
+
+Versions:
+
+- **7B**
+- **13B**
+- **33B**
+- **65B**
+
+!!! attention
+Unfortunately, the open-source model is limited to non-commercial use.
+
+#### 3.2) Open LLaMA
+
+<div style="text-align:center">
+  <img src="https://sapling.ai/static/img/llms/openllama-logo.png" alt="Alt text" width="150"/>
+</div>
+
+In response, a group of students at UC Berkeley in California founded OpenLM Research. Two months later, they released the OpenLLama model based on Meta's LLAMA model.
+
+OpenLLama offers models in versions with **3B**, **7B**, and **13B** parameters. Particularly interesting are the powerful yet "small" V2 models with **3B** and **7B** parameters. These compact yet powerful models were developed from a specially curated dataset.
+
+They are a combination of the Refined Web dataset from Falcon, the Starcoder dataset, and the Redpajama dataset, which is a reproduction of the LLaMA dataset.
+
+<div style="text-align:center">
+  <img src="./assets/os-llms-openllama-dataset.png" alt="Alt text" width="600"/>
+</div>
```

Comment:
In fact, embedding images using HTML `div` containers does not work (at least on my end):

![image](https://github.com/pkeilbach/htwg-practical-nlp/assets/28002199/90f378c3-76b4-467e-9f4a-fc407443883d)

So I suggest to use this schema:

```md
![Open LLaMA dataset](./assets/os-llms-openllama-dataset.png)
```